requirements.txt

```diff
@@ -0,0 +1,7 @@
+lxml
+bleach==1.4.2
+oauthlib==1.0.3
+Mako
+git+https://github.com/edx/XBlock.git#egg=XBlock
```

Comment:
We have a tag in the XBlock repo now.  But pinning versions in a library makes things difficult as we have to change all of them in sync.